backend/src/packages/chaiNNer_standard/image_utility/miscellaneous/change_colormodel.py

```diff
@@ -26,10 +26,10 @@
 
 
 @miscellaneous_group.register(
-    schema_id="chainner:image:change_colorspace",
-    name="Change Colorspace",
+    schema_id="chainner:image:change_colormodel",
+    name="Change Colormodel",
```

Comment:
Maybe this was a mistake and we should have previously separated "color" and "space", but i think here it should for sure be "color model" and not "colormodel". I double checked wikipedia and it has it as two words

Reply:
Updated name to "Change Color Model"